features/duck-player.json

```diff
@@ -0,0 +1,7 @@
+{
+    "_meta": {
+        "description": "Private YouTube video player",
```

Comment:
Could we expand this to be just a bit more descriptive? As this is public facing we're trying to keep these parts clear.

Reply:
Sure! How about now @SlayterDev?